packages/server-wallet/src/__test-with-peers__/wallet/jumpstart.test.ts

```diff
@@ -0,0 +1,95 @@
+import {Wallet} from '../../wallet';
+import {
+  getPeersSetup,
+  messageService,
+  peerEngines,
+  peersTeardown,
+} from '../../../jest/with-peers-setup-teardown';
+import {ObjectiveModel, WalletObjective} from '../../models/objective';
+import {getWithPeersCreateChannelsArgs} from '../utils';
+
+beforeAll(getPeersSetup());
+afterAll(peersTeardown);
+
+jest.setTimeout(60_000);
+
+describe('jumpstartObjectives', () => {
+  it('returns an empty array when there are no objectives', async () => {
+    const wallet = await Wallet.create(peerEngines.a, messageService, {numberOfAttempts: 1});
+
+    const jumpstartResponse = await wallet.jumpStartObjectives();
+    const jumpstartResults = await Promise.all(jumpstartResponse.map(r => r.done));
+
+    expect(jumpstartResults).toHaveLength(0);
```

Comment:
Wait, isn't this equivalent to
```suggestion
    expect(await wallet.jumpStartObjectives()).toHaveLength(0)
```